newreply.php

```diff
@@ -1074,7 +1055,7 @@
 			$post = $db->fetch_array($query);
 			if(!$mybb->user['uid'] || !$post['username'])
 			{
-				$post['username'] = $mybb->get_input('username');
+				$post['username'] = htmlspecialchars_uni($mybb->get_input('username'));
```

Comment:
Please revert this. the `build_postbit` function will handle this on another PR.